### ISSUE TITLE
docker image is blocked on release 0.8.1

### ISSUE THREAD

**Issue by lebuzzer** [Yeicor/rustic-docker#5]
docker image is blocked on release 0.8.1
workflow seems to not working anymore

**Comment by yeicor** [comment]
Yes, I disabled the workflow because it was getting stuck for arm64 builds. I don't have the time to fix it or even use it for now 😅.

I believe it was getting stuck due to extremely slow qemu emulation (this did not happen on previous versions, so maybe it's some kind of bug).

A better solution would be to cross compile the rust code and copy the binary to the image. I can review and merge a pull request if you or any other interested user is up to the task.
